### DISCUSSION TITLE
Icon size in mushroom-template-card

### DISCUSSION THREAD

**Discussion by pille89** [piitaya/lovelace-mushroom#1303] [Q&A]
Hello dear developer I have been searching a lot in forums. But unfortunately I have not found an answer. Is it possible to enlarge the icon in the mushroom-template-card? I set it to vertical and would like the icon to be larger. 

If not, will this be possible at some point?

Thank you very much and really a super frontend integration :-)

**Comment by sanchosk** [comment]
This would be really appreciated, but it seems it did not manage to register on the dev's radar :(

**Comment by BBE-FR** [comment]
You can use card-mod 
https://github.com/thomasloven/lovelace-card-mod

and this forum post to find the good variable to play with mushroom cards:
https://community.home-assistant.io/t/mushroom-cards-card-mod-styling-config-guide/600472

![image](https://github.com/user-attachments/assets/b75d271d-6666-474f-8007-60543c1d88ce)
